### DISCUSSION TITLE
Are `edc.oauth.client.id` & `edc.receiver.http.endpoint` required in the dataplane config?

### DISCUSSION THREAD

**Discussion by florianrusch-zf** [eclipse-tractusx/tractusx-edc#140] [Q&A]
Are the following fields really required for the dataplane? I deployed until now only EDCs with the version 0.1.6 - maybe it has changed in 0.3.0, but I can't believe this.

```properties
edc.oauth.client.id
edc.receiver.http.endpoint
```

I got this information from the README.md from the dataplane modules:
https://github.com/eclipse-tractusx/tractusx-edc/blob/d0b63794056236b79de248af944ffe9bf75cead6/edc-dataplane/edc-dataplane-azure-vault/README.md?plain=1#L22
https://github.com/eclipse-tractusx/tractusx-edc/blob/d0b63794056236b79de248af944ffe9bf75cead6/edc-dataplane/edc-dataplane-hashicorp-vault/README.md?plain=1#L22

**Comment by stephanbcbauer** [comment]
@florianrusch-zf not sure if this is related to #167? In this issue it's mentioned, that the properties are renamed. But I guess your question is related to mandatory or not. Correct?

**Reply by stephanbcbauer** [reply]
@florianrusch-zf is this already answered? If yes, could you please provide an answer?

**Reply by florianrusch-zf** [reply]
Not answered and not solved, as these configurations are still present in the `/edc-dataplane/.../README.md` files.
But it looks to me like these are not needed, since they are not set by our Helm Charts either, for example.

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
